manual/src/main/kotlin/com.r3.corda.finance.manual/types/ManualPayment.kt

```diff
@@ -0,0 +1,18 @@
+package com.r3.corda.finance.manual.types
+
+import com.r3.corda.finance.obligation.types.Payment
+import com.r3.corda.finance.obligation.types.PaymentReference
+import com.r3.corda.finance.obligation.types.PaymentStatus
+import com.r3.corda.sdk.token.money.FiatCurrency
+import net.corda.core.contracts.Amount
+
+/** Represents a manual payment. */
+data class ManualPayment(
+        override val paymentReference: PaymentReference,
+        override val amount: Amount<FiatCurrency>,
```

Comment:
No need to do anything, more a note for me, is that this could be generalised to enable manual delivery of any `TokenType`, e.g. shares.